src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -77,9 +77,9 @@ const useSimpleWebsocket = (url, protocol = undefined) => {
       featLog('=========== CONNECTION CHECKED: true ===============');
       setupWebsocket();
     } catch (e) {
-      console.log('error checking connection');
-      console.log(e);
-      console.log('=========== CONNECTION CHECKED: false ===============');
+      featLog('error checking connection');
+      featLog(e);
+      featLog('=========== CONNECTION CHECKED: false ===============');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the change from `console.log` to `featLog` is a good one for better logging, it's important to consider error handling here. The catch block should not only log the error but also handle it appropriately. If the error is expected and recoverable, you might want to add some logic to retry the connection or provide feedback to the user. If it's an unexpected error, you might want to rethrow it after logging so it can be caught and handled by a higher-level error handler.
```diff
      featLog('=========== CONNECTION CHECKED: true ===============');
      setupWebsocket();
    } catch (e) {
-     featLog('error checking connection');
-     featLog(e);
-     featLog('=========== CONNECTION CHECKED: false ===============');
+     featLog(`Error checking connection: ${e.message}`);
+     // Consider adding error handling logic here
      isInternetConnectedRef.current = false;
      closeSocket();
    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->